osu.Game/Overlays/BeatmapSet/Info.cs

```diff
@@ -120,8 +123,8 @@ public Info()
             {
                 source.Text = b.NewValue?.Source ?? string.Empty;
                 tags.Text = b.NewValue?.Tags ?? string.Empty;
-                genre.Text = b.NewValue?.Genre.Name ?? string.Empty;
-                language.Text = b.NewValue?.Language.Name ?? string.Empty;
+                genre.Text = b.NewValue?.Genre ?? new BeatmapSetOnlineGenre { Id = 1 };
+                language.Text = b.NewValue?.Language ?? new BeatmapSetOnlineLanguage { Id = 1 };
```

Comment:
I think I see what this is trying to do, but I'd probably write it as

```suggestion
                genre.Text = b.NewValue?.Genre ?? new BeatmapSetOnlineGenre { Id = (int)SearchGenre.Unspecified };
                language.Text = b.NewValue?.Language ?? new BeatmapSetOnlineLanguage { Id = (int)SearchLanguage.Unspecified };
```

for less magic constants.